config/application.properties

```diff
@@ -0,0 +1,6 @@
+
+application.info=demo sources for work with files presentation
+java.version=1.8
+
+excel.defaultName=excelFileFromLocationA
+excel.defaultExtension=xls
```

Comment:
For now probably we rid off from keeping both extensions xlx and xlsx, as well as both dependencies. Let's keep to newer one - xlsx.

Reply:
OK